### PR TITLE
Internal: Remove what's new pop-up in playwright tests

### PR DESCRIPTION
**Pull request by dmaziar**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x ] Other... Please describe: playwright test fixes

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
                 "@elementor/externalize-wordpress-assets-webpack-plugin": "0.1.0",
                 "@elementor/extract-i18n-wordpress-experssions-webpack-plugin": "0.1.0",
                 "@elementor/generate-wordpress-asset-file-webpack-plugin": "0.1.0",
-                "@playwright/test": "^1.33.0",
+                "@playwright/test": "1.33.0",
```
Comment (udidol) [inline package-lock.json:50]: I don't think we're supposed to change `package-lock.json`

File: package.json [hunk 2]
```diff
@@ -49,7 +49,7 @@
         "@elementor/externalize-wordpress-assets-webpack-plugin": "0.1.0",
         "@elementor/extract-i18n-wordpress-experssions-webpack-plugin": "0.1.0",
         "@elementor/generate-wordpress-asset-file-webpack-plugin": "0.1.0",
-        "@playwright/test": "^1.33.0",
+        "@playwright/test": "1.33.0",
```
Comment (udidol) [inline package.json:52]: why?

File: package.json [hunk 3]
```diff
@@ -140,4 +140,4 @@
             "react-dom": "$react-dom"
         }
     }
-}
+}
```
Comment (udidol) [inline package.json:143]: empty line?

File: tests/elements-regression/tests/elements-regression.test.js [hunk 4]
```diff
@@ -62,7 +62,7 @@ test.describe( 'Elementor regression tests with templates for CORE', () => {
 				await expect( widget ).not.toHaveClass( /elementor-widget-empty/ );
 				widgetIds.push( id );
 				await editorPage.waitForElementRender( id );
-				await expect( widget ).toHaveScreenshot( `${ widgetType }_${ i }.png`, { maxDiffPixels: 100, timeout: 10000 } );
+				await expect( widget ).toHaveScreenshot( `${ widgetType }_${ i }.png`, { maxDiffPixels: 200, timeout: 10000 } );
```
Comment (udidol) [inline tests/elements-regression/tests/elements-regression.test.js:65]: Why?

**Review by udidol — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (yotamselementor) [reply package.json:52]: We don't want surprises when we run `npm install`, this is a more stable approach

Comment (dmaziar) [reply package.json:52]: also `^` breaks docker to run` test:playwright:elements-regression:linux`

File: package.json (hunk 3 above)
Comment (dmaziar) [reply package.json]: fixed.

File: package-lock.json (hunk 1 above)
Comment (dmaziar) [reply package-lock.json:50]: it is updated automatically on any npm install when someone add/remove/update packages

**Review by udidol — APPROVED** [review]

**Pull request merged**